go/staking/api/sanity_check.go

```diff
@@ -42,12 +42,24 @@ func (p *ConsensusParameters) SanityCheck() error {
 		return fmt.Errorf("minimum commission %v/%v over unity", p.CommissionScheduleRules, CommissionRateDenominator)
 	}
 
+	// Reward schedule steps must be sequential.
+	if len(p.RewardSchedule) > 0 {
+		var prevUntil beacon.EpochTime
+		for _, step := range p.RewardSchedule {
+			if step.Until < prevUntil {
```

Comment:
Equality should probably also be disallowed?

Reply:
```suggestion
			if step.Scale.Cmp(Zero) == -1 || step.Scale.Cmp(RewardAmountDenominator) == 1 {
				fmt.Errorf("reward scale needs to non-negative integer not greater than %d", RewardAmountDenominator)
			}
			if step.Until == beacon.EpochInvalid {
				return fmt.Errorf("invalid epoch")
			}
			if step.Until <= prevUntil {
```

Update tests also.